classes/classroom.rb

```diff
@@ -0,0 +1,15 @@
+# class Classroom
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
```

Comment:
- [ ] In the `Student` class, Please add a method that serves as the 'belongs-to' side (a student belongs to a classroom). Link to the [article with examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)